components/TagPane.tsx

```diff
@@ -0,0 +1,44 @@
+import { SetStateAction, Dispatch, Fragment } from 'react'
+import { useRouter } from 'next/router'
+
+import { FetchedDataSearch, LeafLoader, TagRecordCard } from './'
+import { Details, Paragraph } from '../ui'
+
+import { TranslatedRecordResponse, OrgRecord } from '../types/records'
+import { ENGLISH } from '../constants/language'
+import { useGlobalSearch, useLanguage } from '../hooks'
+import styles from './RecordPane.module.css'
+
+export interface TagPaneProps {
+  orgInfo?: TranslatedRecordResponse
+  setRecords?: Dispatch<SetStateAction<TranslatedRecordResponse>>
```

Comment:
this is unused in the component, so I don't think we need it in the props `interface`